static/css/agency.css

```diff
@@ -245,6 +245,15 @@ fieldset[disabled] .btn-xl.active {
     background-color: #{{ site.data.template.color.secondary }};
 }
 
+.navbar-default.navbar-shrink {
+    padding: 0 0;
+    margin: 0;
+    background-color: #1f3a6d;
+    -webkit-transition: background-color .3s;
```

Comment:
since this is all over the place too, I wonder if we can make it a const so we can just adjust a bunch at once.